frontend/src/screens/dashboard/docsqa/index.tsx

```diff
@@ -204,8 +179,8 @@ const DocsQA = () => {
                   sx={{
                     background: 'white',
                     height: '32px',
-                    width: '13rem',
-                    minWidth: '13rem',
+                    width: '13.2rem',
```

Comment:
Try to use rem values that are equivalent to a whole pixel.

Considering `16px` to be the root size, rem size should be a multiple of `0.0625`